src/decision/decision_engine.h

```diff
@@ -173,21 +174,13 @@ class DecisionEngine {
   /*   return d_needIteSkolemMap.size() > 0; */
   /* } */
 
-  /* add a set of assertions */
-  void addAssertions(const vector<Node> &assertions);
-
   /**
    * add a set of assertions, while providing a mapping between skolem
    * variables and corresponding assertions. It is assumed that all
    * assertions after assertionEnd were generated by ite
    * removal. Hence, iteSkolemMap maps into only these.
    */
```

Comment:
This comment needs to be updated since the mapping is now implicitly given by the assertion pipeline.

Reply:
Done. I've updated the comment and moved the details to assertion_pipeline.h.